.gitignore

```diff
@@ -1,2 +1,6 @@
 Gopkg.lock
 vendor
+
+# IDE Specifics
+
+.idea
```

Comment:
Do you mind not putting IDE specifics here? It gets tricky to track different editors x)

(As a tip for your local environment, you can add ``.idea`` to a [global .gitignore file](https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore)!)